pyproject.toml

```diff
@@ -11,6 +11,7 @@ python-dotenv = "^0.14.0"
 requests = "^2.25.1"
 pytest = "^6.2.3"
 selenium = "^3.141.0"
```

Comment:
You should move pytest and selenium down into the "dev-dependencies" section.

Reply:
Moved to dev-dependencies